### PR TITLE
Gaia module

### PR DESCRIPTION
**Pull request by sblunt**

### CODE REVIEW

**Comment by sblunt** [comment]
This one is built off of `hip-otherinputs`, so once we merge in this branch and that one, we should have a fully functioning Gaia+Hipparcos fit tool!

**Comment by sblunt** [comment]
(issue #231)

File: orbitize/hipparcos.py [hunk 1]
```diff
@@ -390,8 +393,19 @@ def log_prob(model_pars):
...
-
         plt.tight_layout()
         plt.savefig(saveplot, dpi=250)
 
-    return sampler.flatchain, myHipLogProb
\ No newline at end of file
+    return sampler.flatchain, myHipLogProb
+
+
+if __name__ == '__main__':
```
Comment (semaphoreP) [inline orbitize/hipparcos.py:402]: Did you mean to include this snippet with `if __name__ == '_main__':`. Looks like a mini debugging test.

File: orbitize/gaia.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
+import numpy as np
+from astroquery.gaia import Gaia
+from astropy import units as u
+
+class GaiaLogProb(object):
+    """
+    Class to compute the log probability of an orbit with respect to a single
```
Comment (semaphoreP) [inline orbitize/gaia.py:7]: Add a comment somewhere that we are currently only fitting for the position of the star from Gaia, not the proper motion of the Gaia epoch?

File: orbitize/system.py [hunk 3]
```diff
@@ -31,6 +31,7 @@ class System(object):
             IAD fitting. See hipparcos.py for more details.
         fitting_basis (str): the name of the class corresponding to the fitting 
             basis to be used. See basis.py for a list of implemented fitting bases.
+        gaia: TODO
```
Comment (semaphoreP) [inline orbitize/system.py:34]: Is this gaia argument in the docstring filled in in a future PR?

File: orbitize/hipparcos.py (hunk 1 above)
Comment (sblunt) [reply orbitize/hipparcos.py:402]: Nope, I'll remove it. Thanks!

**Review by semaphoreP — APPROVED** [review]
Looks good!

**Pull request merged**